src/main/java/org/sunbird/profile/service/UserBulkUploadService.java

```diff
@@ -209,6 +211,9 @@ private List<String> validateEmailContactAndDomain(UserRegistration userRegistra
         if (!ProjectUtil.validateContactPattern(userRegistration.getPhone())) {
             errList.add("Invalid Contact Number");
         }
+        if (!userUtilityService.validatePosition(userRegistration.getPosition())) {
+            errList.add("Invalid Position");
+        }
```

Comment:
You need to check the position is null or not first...
if the position is null - the current code will still try to call DB -- which will throw error.. 
as mentioned previous - if position is null - then error should be - Positions is missing.